float32 yaw
float32 dist
bool jump
---
bool state
